Library/Homebrew/language/haskell.rb

```diff
@@ -92,5 +92,27 @@ def install_cabal_package(*args, **options)
         end
       end
     end
+    module Stack
+      # run stack install, using the current latest LTS snapshot on stackage.org
+      def stack_install(*args)
+        system "stack", "install", "--resolver=lts", *args
+      end
+
+      # install tools, sequentially as some tools can depend on other tools
+      def stack_install_tools(*tools)
+        tools.each { |tool| stack_install tool }
+      end
+
+      def install_stack_package(*args, **options)
+        stack_install_tools(*options[:using]) if options[:using]
```

Comment:
I don't think this "tools" thing (or the `:using` option) is necessary for `stack`, as it will automatically install any needed tools locally.